.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Help & Support
+    url: https://community.cryptomator.org/
+    about: You will find answers in our community forum
```

Comment:
Should we link to a new Hub section in the forum?

Reply:
Or maybe delete this since it's a "closed" beta?

Reply:
Well for now the GH interaction limit is in place, so you won't see the issue template chooser at all, I guess.

Reply:
We could indeed change the URL to https://community.cryptomator.org/tag/hub but I personally prefer the category view and would wait and see what and how we finally want to offer support for Hub. In our other repos this is also not set and our community seems to get along with it...